apps/vr-tests-react-components/src/stories/Toast/Toast.stories.tsx

```diff
@@ -11,6 +11,7 @@ import {
 } from '@fluentui/react-toast';
```

Comment:
## [🕵 fluentuiv9 Open the Visual Regressions report to inspect the 30 screenshots]( https://staticpagerelease.z16.web.core.windows.net/azurestaticapps/vr-fluent/index.html?organization=uifabric&project=fabricpublic&prId=28401&commitId=e8d4c42ac5d54dbae7d4b78a6a545fb7ef660342&env=prod)

✅ There was 30 screenshots added, 0 screenshots removed, 2010 screenshots unchanged, 0 screenshots with different dimensions and 0 screenshots with visible difference.


  <details><summary><b>unknown</b> 30 screenshots</summary>

|Image Name|Diff(in Pixels)| Image Type
|-|-|-|
|Toast.Full Toast - Dark Mode.Toast visible.chromium.png|0| Added|
|Toast.Full Toast - Dark Mode.chromium.png|0| Added|
|Toast.Full Toast - High Contrast.Toast visible.chromium.png|0| Added|
|Toast.Full Toast - High Contrast.chromium.png|0| Added|
|Toast.Full Toast - RTL.Toast visible.chromium.png|0| Added|
|Toast.Full Toast - RTL.chromium.png|0| Added|
|Toast.Full Toast Inverted - Dark Mode.Toast visible.chromium.png|0| Added|
|Toast.Full Toast Inverted - Dark Mode.chromium.png|0| Added|
|Toast.Full Toast Inverted - High Contrast.Toast visible.chromium.png|0| Added|
|Toast.Full Toast Inverted - High Contrast.chromium.png|0| Added|
|Toast.Title Only - Dark Mode.Toast visible.chromium.png|0| Added|
|Toast.Title Only - Dark Mode.chromium.png|0| Added|
|Toast.Title Only - High Contrast.Toast visible.chromium.png|0| Added|
|Toast.Title Only - High Contrast.chromium.png|0| Added|
|Toast.Title Only - RTL.Toast visible.chromium.png|0| Added|
|Toast.Title Only - RTL.chromium.png|0| Added|
|Toast.Title Only Inverted - Dark Mode.Toast visible.chromium.png|0| Added|
|Toast.Title Only Inverted - Dark Mode.chromium.png|0| Added|
|Toast.Title Only Inverted - High Contrast.Toast visible.chromium.png|0| Added|
|Toast.Title Only Inverted - High Contrast.chromium.png|0| Added|
|Toast.Without Subtitle - Dark Mode.Toast visible.chromium.png|0| Added|
|Toast.Without Subtitle - Dark Mode.chromium.png|0| Added|
|Toast.Without Subtitle - High Contrast.Toast visible.chromium.png|0| Added|
|Toast.Without Subtitle - High Contrast.chromium.png|0| Added|
|Toast.Without Subtitle - RTL.Toast visible.chromium.png|0| Added|
|Toast.Without Subtitle - RTL.chromium.png|0| Added|
|Toast.Without Subtitle Inverted - Dark Mode.Toast visible.chromium.png|0| Added|
|Toast.Without Subtitle Inverted - Dark Mode.chromium.png|0| Added|
|Toast.Without Subtitle Inverted - High Contrast.Toast visible.chromium.png|0| Added|
|Toast.Without Subtitle Inverted - High Contrast.chromium.png|0| Added|

</details>


<div id="vrtCommentfluentuiv9"/>